### PR TITLE
Test/dialog box

### PR DESCRIPTION
**Pull request by 444B**
# Pull Request Template

## TL;DR
Provide a brief summary of the changes made.
- shows the display results inside a dialogbox

## Problem Solved
1. Describe the problem your changes address. 
2. Please mention the issue number if relevant
- ease of use and reduced having to scroll down to bottom of the page to see results

## Screenshots
If applicable, add screenshots to help explain your solution. For adding images, use:
- 

## Changes Made(Optional)
List the main changes in bullet points.

- [ ] results now show in a dialogbox

## Manual Testing
- [X] You have manually tested all changes to ensure they work as intended. Replace the [ ] with a [X] if True.

**How was this tested?**
Describe the tests that you ran to verify your changes.
- uploaded to testpypi and run locally. Can ignore failed test package since I have to push a new version with a typo fix

## Additional Notes
Include any additional information that might be relevant for reviewers.
-

### CODE REVIEW

**Comment by 444B** [comment]
Failed GH action WF can be ignored since I had to push a new commit with an existing 8.0.0 version and testpypi wont accept duplicate numbers. This will be accepted in main branch
